guides/v3.2.0/tutorial/autocomplete-component.md

```diff
@@ -229,14 +229,14 @@ export default Controller.extend({
   actions: {
     filterByCity(param) {
       if (param !== '') {
-        return this.get('store').query('rental', { city: param });
-        return this.get('store')
+        return this.store.query('rental', { city: param });
+        return this.store
```

Comment:
It might just be the formatting here on my phone, but I'm not even sure I understand what's happening in these lines. Is it two return statements in a row? The second one would never be reached, right? If so I think it'd be good to quickly clean it up as part of this PR.

Reply:
good catch, this needs to be fixed, will do.

Reply:
this is displayed as diff in the rendered version: https://guides.emberjs.com/release/tutorial/autocomplete-component/#toc_handling-results-coming-back-at-different-times
the redundant code makes sense here.